docs/manifest/schema.rst

```diff
@@ -81,3 +83,22 @@ Manifest Schema
             arguments:   # arguments that override the global script arguments
                          # (optional)
                 <key>: <value>
+
+            max_runtime: HH:MM,   # max runtime in hours:minutes (optional)
+
+            interest_list:       # list of email addresses (optional)
+                - list@domain
+                - user@domain
+
+            priority: #  job priority (integer) (optional)
+
+            testbed:   # (optional)
+                name:  # name of the testbed
+
+            topology:  # (optional)
+                name:  # name of the topology
+
+            docker:  # docker details  (optional)
+                volumes:  # docker volume mounts
```

Comment:
i feel this could lead to more questions 

Is there more documentation on this?

Reply:
No docs on xpresso other than mention of "Cloud Mount Points" under job execution